pages/linux/eww.md

```diff
@@ -0,0 +1,28 @@
+# eww
+
+> Implement your own custom widgets in any window manager.
+> More information: <https://elkowar.github.io/eww>.
+
+- Start the daemon:
+
+`eww daemon`
+
+- Open a widget:
+
+`eww open {{path/to/source_code_directory}}`
+
+- Close a widget:
+
+`eww close {{path/to/source_code_directory}}`
```

Comment:
Is this correct? The documentation says the argument should be a window name.
https://elkowar.github.io/eww/eww.html#running-eww
> Then to run it, enter:
> ```
> ./eww daemon
> ./eww open <window_name>
> ```

```suggestion
- Open a widget:

`eww open {{window_name}}`

- Close a widget:

`eww close {{window_name}}`
```

Reply:
Thanks for catching that! I just realized I didn't put the full thing.